compose.yml

```diff
@@ -119,6 +122,14 @@ services:
             - MATOMO_DATABASE_PASSWORD=$MW_DB_INSTALLDB_PASS
         volumes:
             - ./_data/matomo:/var/www/html
+        labels:
+            - "traefik.enable=true"
+            - "traefik.http.routers.matomo.rule=Host(`bugsigdb-staging.wikiworks.com`) && PathPrefix(`/matomo`)"
+            - "traefik.http.routers.matomo.entrypoints=websecure"
+            - "traefik.http.middlewares.matomo-stripprefix.stripprefix.prefixes=/matomo"
+            - "traefik.http.routers.${TRAEFIK_STACK_PREFIX}-matomo.middlewares=${TRAEFIK_STACK_PREFIX}-matomo-stripprefix"
```

Comment:
>  but at this point it won't work because the location of matomo is at /var/www/html and in order to work need to be moved to /var/www/html/matomo

It does not work because there is no defined middleware with name `${TRAEFIK_STACK_PREFIX}-matomo-stripprefix` (only `matomo-stripprefix` which differs)